src/app/pages/system/CloudCredentials/cloudcredentials-form/cloudcredentials-form.component.ts

```diff
@@ -47,6 +47,12 @@ export class CloudCredentialsFormComponent {
       type: 'input',
       name: 'client_id-AMAZON_CLOUD_DRIVE',
       placeholder: T('Amazon Application Client ID'),
+      tooltip:T('Client ID for the <a\
+                 href="https://developer.amazon.com/docs/amazon-drive/ad-get-started.html"\
+                 target="_blank">Amazon Drive account</a>. Note: <a\
+                 href="https://rclone.org/amazonclouddrive/"\
+                 target="_blank">rclone</a> only supports Amazon Drive\
+                 accounts that have an existing set of API keys.'),
```

Comment:
What does this line mean?  Existing where?  "that have" is kind of ambiguous, too.